smarts/core/sumo_traffic_simulation.py

```diff
@@ -266,7 +271,6 @@ def teardown(self):
             return
 
         assert self._is_setup
-        self._close_traci_and_pipes()
```

Comment:
If you are removing the traci close guarantee here I would suggest adding a destroy method to make sure that the pipes get closed.

Reply:
Actually `initialize_traci_conn` will call `_close_traci_and_pipes` so I think the pipes are not leaking.